tests/runtime/builtin

```diff
@@ -185,14 +185,14 @@ EXPECT ^([0-9]+\.[0-9]+ ?)+.*$
 TIMEOUT 5
 
 NAME log size too small
-ENV BPFTRACE_LOG_SIZE=1
-RUN {{BPFTRACE}} -ve 'BEGIN { if (str($1) == str($2)) { printf("%s\n", str($1)); exit() } }' "hello" "hello"
+ENV BPFTRACE_LOG_SIZE=0
+RUN {{BPFTRACE}} -e 'BEGIN { if (str($1) == str($2)) { printf("%s\n", str($1)); exit() } }' "hello" "hello"
```

Comment:
Why don't we change the expected error message?

Reply:
Tried changing that, but CI is failing. Reference commit [5a8ea28] (first one in series)

Reply:
Do you have a link to the failed CI job? Or could you retry it so that we can have a look on the failure?

FWIW, I locally ran `.github/include/ci.py` with that commit and the test passed.

Reply:
For me .github/include/ci.py produced failure with current changes:

[  FAILED  ] builtin.log size too small
	Command: ../src//bpftrace -e 'BEGIN { if (str($1) == str($2)) { printf("%s\n", str($1)); exit() } }' "hello" "hello"
	Expected: Error loading program: BEGIN
	Found: Attaching 1 probe...\n__BPFTRACE_NOTIFY_PROBES_ATTACHED\nhello\n\n\n

Reply:
I took the liberty and reformatted your answer to make it more readable.

> "Error: Failed to load program" string occurs properly under -v option but the expected string is not occurring without -v option.

That is expected, isn't it? What I'm suggesting is to keep the execution options as they are (`-ve`) but change the expected message to match the string appearing under `-v`.

This patch works for me locally:
```diff
diff --git a/tests/runtime/builtin b/tests/runtime/builtin
index 77efbdd1..18b2b305 100644
--- a/tests/runtime/builtin
+++ b/tests/runtime/builtin
@@ -187,7 +187,7 @@ TIMEOUT 5
 NAME log size too small
 ENV BPFTRACE_LOG_SIZE=1
 RUN {{BPFTRACE}} -ve 'BEGIN { if (str($1) == str($2)) { printf("%s\n", str($1)); exit() } }' "hello" "hello"
-EXPECT Error loading program: BEGIN
+EXPECT Error: Failed to load program
 TIMEOUT 5
 WILL_FAIL
```

Reply:
Done changes. The suggested solution works well on my local system also, but CI fails with same.

Reply:
Thanks @dubeyabhishek.

The test in the CI fails with:
```
[  FAILED  ] builtin.log size too small
	Command: ../src//bpftrace -ve 'BEGIN { if (str($1) == str($2)) { printf("%s\n", str($1)); exit() } }' "hello" "hello"
	Expected: Error: Failed to load program
	Found: INFO: node count: 16\nAttaching 1 probe...\n\nError log: \n\nERROR: Error loading program: BEGIN\n
```
which is quite interesting b/c it seems that the `-v` option is ignored in CI. It's not that important to spend more time debugging it, so let's use your original solution (dropping `-v` and setting log size to 0).

Thanks!